src/missile.cpp

```diff
@@ -18,7 +18,7 @@ void Missile::draw(GraphicsHandler *graphics) {
         printf("Missile graphics were null! Setting missile graphic...\n");
         set_texture(graphics->load_image("assets/projectiles/missile.png"));
     }
-    graphics->draw(missile_texture_, (int)x_pos(), (int)y_pos());
+    graphics->draw(missile_texture_, (int)x_pos(), (int)y_pos(), GraphicPriority::MIDDLE);
```

Comment:
I was thinking, that maybe we should push for something playable first. Then do a big cleanup like adding using a config file, but having something playable first might help motivate us better.